packages/cms/src/migrations/sprint18/index.ts

```diff
@@ -0,0 +1,96 @@
+import client from 'part:@sanity/base/client';
+
+/**
+ * This migration first checks if any 'veelgesteldeVragenGroups' documents exist, if not, it creates one called 'Algemeen/General'.
+ * The first of the 'veelgesteldeVragenGroups' documents is then retrieved.
+ *
+ * Then it changes the '_type' field for all 'veelgesteldeVragen.question' documents from 'collapsible' to 'faqQuestion' and
+ * assigns the veelgesteldeVragenGroups document to the 'group' field.
+ *
+ * This migration can be run by executing the following command from the packages/cms directory:
+ * sanity exec src/migrations/sprint18 --with-user-token
+ *
+ * (Don't forget to run 'sanity login' first and choose the 'Google' option to login with)
```

Comment:
```suggestion
 * (Don't forget to run 'sanity login' first)
```

Not strictly speaking necessary to use that

Reply:
True, but in our case, it is since all of our logins are associated with our Google accounts.